technical-reports/format/groups.md

```diff
@@ -164,6 +158,46 @@ For example:
 
 </aside>
 
+### Extensions
+
+Groups MAY include an optional `$extensions` property where tools MAY add proprietary, user-, team- or vendor-specific data to a group. When doing so, each tool MUST use a vendor-specific key whose value MAY be any valid JSON data.
+
+Group extensions follow the same rules as [design token extensions](#extensions), the only difference is that they relate to the group itself.
+
+- The keys SHOULD be chosen such that they avoid the likelihood of a naming clash with another vendor's data. The [reverse domain name notation](https://en.wikipedia.org/wiki/Reverse_domain_name_notation) is recommended for this purpose.
+- Tools that process design token files MUST preserve any extension data they do not themselves understand. For example, if a group contains extension data from tool A and the file containing that data is opened by tool B, then tool B MUST include the original tool A extension data whenever it saves a new design token file containing that group.
+
+Note that, since a group's `$extensions` only relate to that group, they do not have any effect on nested groups or tokens. However, if an extension author wants their extension to be inherited by nested groups or tokens like a group's `$type` is, then they MAY implement that behaviour in whatever tools support their extension.
```

Comment:
@c1rrus I'm concerned about the ambiguity here. If one vendor's `$extensions` are inherited by children but another vendor's aren't, I'm concerned about our ability to "pave cowpaths" in the future.

My expectation would be that group `$extensions` are *not* inherited by child groups or tokens.

Reply:
This also seems to be the general consensus in the original issue: #106.

I'd delete the sentence that starts with "However, if an extension author..." and call this one done. 

PR looks good otherwise.